File: portfolio/quay/back_end/payments2/mercadopago/api-mercadopago-master/templates/code-examples-master/customized-checkout/payment/others/bank_transfer/php/create_payment.php
<?php
require_once ('mercadopago.php');

$mp = new MP('ACCESS_TOKEN');

$payment_data = array(
	"transaction_amount" => 100,
	"description" => "Title of what you are paying for",
	"payer" => array (
		"email" => "test_user_19653727@testuser.com",
		"identification" => array(
			"type" => "IDENTIFICATION_TYPE_ID",
			"number" => "12345678"
		)
	),
	"payment_method_id" => "webpay",
	"transaction_details" => array(
		"financial_institution" => 1234
	),
	"additional_info" => array(
		"ip_address" => "127.0.0.1"
	),
	"callback_url" => "http://www.your-site.com"
);

$payment = $mp->post("/v1/payments", $payment_data);

File: portfolio/quay/back_end/payments2/mercadopago/api-mercadopago-master/templates/code-examples-master/customized-checkout/payment/one-click/python/create_payment.py
import mercadopago
import json

mp = mercadopago.MP("ACCESS_TOKEN")

payment = mp.post("/v1/payments", {
        "transaction_amount": 100,
        "token": "ff8080814c11e237014c1ff593b57b4d",
        "description": "Title of what you are paying for",
        "installments": 1,
        "payer": {
            "type": "customer",
            "id": "12345678-abcdefghij"
        }
    });

print(json.dumps(payment, indent=4))
